bson/_cbsonmodule.c

```diff
@@ -223,8 +223,8 @@ static long long millis_from_datetime_ms(PyObject* dt){
         }
     }
     Py_DECREF(ll_millis);
-
-    return millis;
+    *out = millis;
+    return 0;
```

Comment:
Nice! One followup nit: Can you refactor this method to return 0 on error and 1 on success? That's the pattern we use for all the other methods.